aptos-move/framework/aptos-framework/sources/reconfiguration.move

```diff
@@ -150,4 +150,25 @@ module aptos_framework::reconfiguration {
             },
         );
     }
+
+    // For tests, skips the guid validation.
```

Comment:
Wasn't sure how to get the GUID to equal 1 during testing so had to force create one for `aptos_framework`.

Reply:
This is fine 👍